pages/common/nmap.md

```diff
@@ -11,18 +11,18 @@
 
 `nmap {{ip_or_hostname}} {{optional_another_address}}`
 
-- Also enable service detection, OS fingerprinting and traceroute.
+- Also enable scripts, service detection, OS fingerprinting and traceroute.
 
 `nmap -A {{address_or_addresses}}`
 
-- Assume good network connection and speed up execution:
+- Assume good network connection and speed up execution.
 
 `nmap -T4 {{address_or_addresses}}`
 
-- Scan a specific list of ports (use -p- for all ports 1-65535)
+- Scan a specific list of ports (use -p- for all ports 1-65535).
 
 `nmap -p {{port1,port2,...,portN}} {{address_or_addresses}}`
 
-- Perform TCP and UDP scanning (use -sU for UDP only, -sZ for SCTP, -sO for IP):
+- Perform TCP and UDP scanning (use -sU for UDP only, -sZ for SCTP, -sO for IP).
```

Comment:
Don't use this trick to have +1 code example in parens.
If the example is common and useful add it separately.